src/typed-method-types/workflows/triggers/webhook.ts

```diff
@@ -11,3 +18,20 @@ export type WebhookTrigger =
       filter?: FilterType;
     };
   };
+export type WebhookTriggerResponse<TriggerDefinition extends WebhookTrigger> =
+  Promise<
+    WebhookResponse<TriggerDefinition> | FailedTriggerResponse
+  >;
+
+export type WebhookResponse<TriggerDefinition extends WebhookTrigger> =
+  & BaseResponse
+  & {
+    trigger: WebhookTriggerObject<TriggerDefinition>;
+  };
+
+export type WebhookTriggerObject<TriggerDefinition extends WebhookTrigger> =
+  & BaseTriggerResponse<TriggerDefinition>
+  & {
+    webhook?: TriggerDefinition["webhook"];
```

Comment:
Does the response ever not include a `webhook` property for webhook triggers? Seems like it should